tests/export_test.py

```diff
@@ -38,6 +42,17 @@
 
 config.parse_flags_with_absl()
 
+prev_xla_flags = None
+def setUpModule():
+  global prev_xla_flags
+  # This will control the CPU devices. On TPU we always have 2 devices
+  prev_xla_flags = jtu.set_host_platform_device_count(2)
+
+# Reset to previous configuration in case other test modules will be run.
+def tearDownModule():
+  prev_xla_flags()
```

Comment:
What does it mean to call a variable?

Reply:
This variable holds a cleanup function. We use the same code helper functions and style in, e.g., debugging_test.py

Reply:
Ohh, ok. This is a very weird and dangerous pattern lol. But it's fine.